distributed/protocol/core.py

```diff
@@ -15,6 +21,39 @@
 
 logger = logging.getLogger(__name__)
 
+cache_dumps = LRU(maxsize=100)
+cache_loads = LRU(maxsize=100)
+cache_dumps_lock = threading.Lock()
+cache_loads_lock = threading.Lock()
```

Comment:
Since we are discussing getting rid of `dumps_function`, will these still be needed or will they go away as well?